plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -148,6 +151,10 @@ public ResultSet getTables(Connection connection, Optional<String> schemaName, O
                     return Optional.of(varcharColumnMapping(createUnboundedVarcharType(), true));
                 }
                 return Optional.of(defaultVarcharColumnMapping(columnSize, true));
+            case Types.TIMESTAMP:
+                TimestampType timestampType = createTimestampType(3);
+                // TODO: use {@link #timestampColumnMapping} when https://issues.apache.org/jira/browse/CALCITE-1630 gets resolved
+                return Optional.of(timestampColumnMappingUsingSqlTimestampWithRounding(timestampType));
```

Comment:
`timestampColumnMappingUsingSqlTimestampWithRounding()` has been deprecated. Replace it with `timestampColumnMapping()` available in same `StandardColumnMappings`.

Reply:
@sumannewton Since Avatica doesn't support `LocalDateTime`, we have to use `timestampColumnMappingUsingSqlTimestampWithRounding`: [related discussions](https://github.com/trinodb/trino/issues/8404#issuecomment-874597612)